charts/operator-wandb/charts/parquet/templates/cron.yaml

```diff
@@ -0,0 +1,149 @@
+{{- if .Values.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "cron.fullname" . }}
+  labels:
+    {{- include "wandb.commonLabels" . | nindent 4 }}
+    {{- include "parquet.commonLabels" . | nindent 4 }}
+    {{- include "parquet.labels" . | nindent 4 }}
+    {{- if .Values.deployment.labels -}}
+    {{-   toYaml .Values.deployment.labels | nindent 4 }}
+    {{- end }}
+  annotations:
+    {{- include "wandb.deploymentAnnotations" $ | nindent 4 }}
+    {{- if .Values.deployment.annotations -}}
+    {{-   toYaml .Values.deployment.annotations | nindent 4 }}
+    {{- end }}
+spec:
+  schedule: "{{ .Values.backfillSchedule }}"
+  concurrencyPolicy: {{ .Values.concurrencyPolicy }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: {{ include "cron.name" . }}
+              image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+              command: 
+                {{ toYaml .Values.backfillCommand | nindent 16 }}
```

Comment:
Is this not a static command?

Reply:
This was changed, this is not a templated command anymore.